tests/interfaces/test_numpyro/test_numpyro_marginal_ll.py

```diff
@@ -0,0 +1,292 @@
+import chex
+import jax.numpy as jnp
+import jax.random as jr
+import numpyro
+import numpyro.distributions as dist
+import pytest
+from numpyro.contrib.tfp import distributions as tfd
+from numpyro.distributions import constraints
+
+from gpjax.gps import Prior
+from gpjax.interfaces.numpyro import (
+    add_constraints,
+    add_priors,
+    numpyro_dict_params,
+    numpyro_marginal_ll,
+)
+from gpjax.kernels import RBF
+from gpjax.likelihoods import Gaussian
+from gpjax.parameters import initialise
+
+seed = 123
+KEY = jr.PRNGKey(seed)
+
+# TODO: test conjugate posterior
+def _get_conjugate_posterior_params() -> dict:
+    kernel = RBF()
+    prior = Prior(kernel=kernel)
+    lik = Gaussian()
+    posterior = prior * lik
+    params = initialise(posterior)
+    return params, posterior
+
+
+def _gen_training_data(n_samples, n_features, n_latents):
+
+    x = jr.normal(key=KEY, shape=(n_samples, n_features))
+    y = jr.normal(key=KEY, shape=(n_samples, n_latents))
+    return x, y
+
+
+def test_numpyro_marginal_ll_params():
+
+    # create sample data
+    X, y = _gen_training_data(10, 10, 2)
+
```

Comment:
Line here to instantiate `Dataset` using `X` and `y.

Reply:
Fixed!